app.js

```diff
@@ -1,11 +1,64 @@
 /* Imports */
+const shell1Button = document.getElementById('shell1-button');
+const shell2Button = document.getElementById('shell2-button');
+const shell3Button = document.getElementById('shell3-button');
+
+const shell1Container = document.getElementById('shell1-container');
+const shell2Container = document.getElementById('shell2-container');
+const shell3Container = document.getElementById('shell3-container');
+
+const totalEl = document.getElementById('total');
+const lossesEl = document.getElementById('losses');
+const winsEl = document.getElementById('wins');
+
+let correctGuesses = 0;
+let totalGuesses = 0;
 
 /* Get DOM Elements */
+shell1Button.addEventListener('click', () => {
+    const correctSpot = getRandomHidingSpot();
+    handleGuess('shell1', correctSpot);
+    console.log(correctSpot);
+});
 
-/* State */
+shell2Button.addEventListener('click', () => {
+    const correctSpot = getRandomHidingSpot();
+    handleGuess('shell2', correctSpot);
+    console.log(correctSpot);
+});
 
+shell3Button.addEventListener('click', () => {
+    const correctSpot = getRandomHidingSpot();
+    handleGuess('shell3', correctSpot);
+    console.log(correctSpot);
+});
+
+/* State */
+function getRandomHidingSpot() {
+    const hidingPlaces = ['shell1', 'shell2', 'shell3'];
+    const index = Math.floor(Math.random() * hidingPlaces.length);
+    const correctSpot = hidingPlaces[index];
+    console.log(correctSpot);
+    return correctSpot;
+}
 /* Events */
+function handleGuess(userGuess, correctSpot) {
+    shell1Container.classList.remove('assets/shell-b.png');
```

Comment:
lines 46-48 here could be refactored and put in a new function called resetStyles().

then call the function here in the handleGuess() function on line 46